### ISSUE TITLE
Europe Affiliates shall have the opportunity to post to the timeline by email

### ISSUE THREAD

**Issue by mattl** [creativecommons/europe#5]
(in addition to twitter and WP login), and for that a dedicated email address like cc-europe-timeline@creativecommons.org would be great. If possible it would make sense to limit the this address to only receive/filter emails from those addresses that are on the cc-europe mailing list.

**Comment by mattl** [comment]
This isn't something we've setup for other sites, and it would be a lot of work to set it up and maintain it. Instead we should ensure Affiliates can login using their CCID account, rather than maintaining another login for a website.

**Comment by RCEurope1** [comment]
maybe there was a misunderstanding: There's no separate login required for the post-by-mail functionality. And, this feature was already set up within the Wordpress by Michael (using the Postie Plugin), only the dedicated email account to receive the emails needs to be set up. Please set up such an address and provide the credentials to Gwen and me. thx

**Comment by mattl** [comment]
Yeah, that's not something we're going to offer. I don't want to expose a CC Mailbox in this manner.

**Comment by RCEurope1** [comment]
expose to what exactly?
